src/VerseSuggesting.ts

```diff
@@ -38,47 +39,9 @@ export class VerseSuggesting implements IVerseSuggesting {
    * @constructor
    */
   public get ReplacementContent(): string {
-    let head = `> [!Bible]`
-    let bottom = ''
-    if (this.settings?.collapsibleVerses) {
-      head += '-'
-    }
-    if (
-      this.settings.referenceLinkPosition ===
-        BibleVerseReferenceLinkPosition.Header ||
-      this.settings.referenceLinkPosition ===
-        BibleVerseReferenceLinkPosition.AllAbove
-    ) {
-      head += this.getVerseReference()
-    }
-    if (
-      this.settings.referenceLinkPosition ===
-        BibleVerseReferenceLinkPosition.Bottom ||
-      this.settings.referenceLinkPosition ===
-        BibleVerseReferenceLinkPosition.AllAbove
-    ) {
-      bottom += `> \n ${this.getVerseReference()}`
-    }
+    let head = this.formatReferenceString(this.settings.headFormatString)
```

Comment:
I belive the broken change is not necessary, it could be converted from old setting to new template settings without broken it, if change it like this

```suggestion
      if (this.settings?.collapsibleVerses) {
      oldhead += '-'
    }
    if (
      this.settings.referenceLinkPosition ===
        BibleVerseReferenceLinkPosition.Header ||
      this.settings.referenceLinkPosition ===
        BibleVerseReferenceLinkPosition.AllAbove
    ) {
      oldhead += ' {{verse_reference_link}}'
    }
    let head = this.formatReferenceString(this.settings.headFormatString || oldhead)
```

Reply:
This is what I mean in the last comment the settings should be translateble between old and new.


Ref: https://github.com/tim-hub/obsidian-bible-reference/pull/109#discussion_r1299301939

Reply:
I am unfamiliar with the `string || string` syntax and can't seem to find a reference through Google, is there a link to how that should behave somewhere?